src/select.html

```diff
@@ -13,4 +13,7 @@
 	<li ng-mousedown="updateModel(item)" ng-repeat="item in ::collection" ng-class="{'+selected': (selectedIndex === $index)}">
 		<a>{{appendText}} {{isString(item.value) ? item.value : item}}</a>
 	</li>
+	<li ng-if="linkText && linkHref">
+		<a href="{{linkHref}}" class="cps-blue-link">{{linkText}}</a>
+	</li>
```

Comment:
It does not screw up the display. You cannot highlight with up/down keys since it is not truly an option, it is just a link. It must be clicked, you can't highlight it and "choose" it.